server/app/controllers/admin/AdminExportController.java

```diff
@@ -75,6 +75,9 @@ public Result index(Http.Request request, Long programId) {
       return badRequest(String.format("Program with ID %s could not be found", programId));
     }
 
+    // TODO(#7087) migrate program categories
+    program = program.toBuilder().setCategories(ImmutableList.of()).build();
```

Comment:
We want this to run whether or not the filtering feature flag is on in case admins toggled the flag on, added categories, and then toggled it off again.